.circleci/config.yml

```diff
@@ -319,6 +319,15 @@ defaults:
         CXX: clang++
         MAKEFLAGS: -j 2
 
+  - base_ubuntu2004_clang_medium: &base_ubuntu2004_clang_medium
```

Comment:
You can delete this and just rename the `small` resource class to `medium`, and set its `resource_class` to `medium`, since it isn't used anywhere anymore.